### PR TITLE
Add OkHttp Plugin

### PR DESCRIPTION
**Pull request by ITGrocery**
Motivation:
Hello,I am a java developer of ehi .In our team,we also use opentracing to collect tracing info and send it to elk.Our team use okhttp but not httpclient and resttemplate,so I want to add the okhttp plugin.

Modification:
I add two submodules ,the one is okhttp-plugin,the two is sample-with-okhttp.

Result:
The okhttp plugin will trace the info of remote call when your project using the okhttp library.

### CODE REVIEW

**Comment by ITGrocery** [comment]
Why the Travis CI prints this error? How to solve it?

**Comment by glmapper** [comment]
> Why the Travis CI prints this error? How to solve it?
> 

You can try mvn compile locally and resubmit

**Comment by ITGrocery** [comment]
> > Why the Travis CI prints this error? How to solve it?
> > 
> 
> You can try mvn compile locally and resubmit

I compiled locally and successfully.

File: sofa-tracer-plugins/sofa-tracer-okhttp-plugin/pom.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    <dependencies>
+        <dependency>
+            <groupId>com.alipay.sofa</groupId>
+            <artifactId>tracer-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp</artifactId>
+            <version>3.8.1</version>
```
[the remark this answers is not in the document]
Comment (ITGrocery) [reply sofa-tracer-plugins/sofa-tracer-okhttp-plugin/pom.xml]: Ok,I have updated okhttp version and removed not tested code , finally all checks have passed.
I have a question and want to ask you ,if my code don't fit your code style , the console will print  un commit files error , I think the cause is that the maven plugin will format my code, so the check_format.sh will exit with -1 .I imported your Formatter.xml into IDEA , and found the cod e style don't like yours .

**Pull request merged**